layers/README.md

```diff
@@ -119,3 +129,52 @@ lo4 = LayerOps(score=lambda x: '; '.join(x),
 out_layer6 = lo4.process([demo2, demo3])                      # Trigger processing on a list of demo2 and demo0
 out_layer6.to_file("C:\demo_layer6.json")                     # Save combined comment layer to file
 ```
+
+## to_excel.py
+to_excel.py provides the ToExcel class, which is a way to export an existing layer file as an Excel 
+spreadsheet. The ToExcel class has an optional parameter for the initialization function, that 
+tells the exporter to build the output matrix based on live data from cti-taxii.mitre.org. It 
+can also use cached subtechniques data. If neither of these is used, it will default to pulling from the 
+ATT&CK cti repo. If matrix template lacks elements included in a layer's technique listing, those 
+elements will not be visible in the output file.
+
+##### ToExcel()
+```python
+x = ToExcel(domain='enterprise', server=False, local=None)
+```
+The ToExcel constructor takes domain, server, and local arguments during instantiation. The domain can 
+be either 'enterprise' or 'mobile', and can be pulled from a layer file as `layer.domain`. The server 
+argument tells the matrix generation tool to use the `cti-taxii` server when building the matrix. The 
+local argument tells the matrix generation tool the path to a local stix repository to use when 
+constructing the matrix. 
+
+##### .to_file() Method
+```python
+x.to_xlsx(layer=layer, filepath="layer.xlsx")
+```
+The to_xlsx method exports the layer file referenced as `layer`, as an excel file to the 
+`filepath` specified. 
+
+#### Example Usage
+```python
+from layers import Layer
+from layers import ToExcel
+
+
+lay3 = Layer()
+lay3.from_file('layers/heatmap_example.json')
+exporter = ToExcel(domain=lay3.layer.domain, server=False, local=None)
+exporter.to_xlsx(layer=lay3, filepath="layer.xlsx")
+
+lay = Layer()
+lay.from_file("path/to/layer/file.json")
+# Using cached data for template
+t = ToExcel(domain=lay.layer.domain, server=False, local=None)
+t.to_xlsx(layer=lay, filepath="demo.xlsx")
+# Using live data for templates
+t2 = ToExcel(domain='enterprise', server=True, local=None)
+t.to_xlsx(layer=lay, filepath="demo2.xlsx")
+#Using cached local data for templates
+t3 = ToExcel(domain='mobile', server=False, local='path/to/local/stix')
+t3.to_xlsx(layer=lay, filepath="demo3.xlsx")
```

Comment:
> #Using cached data for template

It's not, though? It's using data on MITRE/CTI, which is not local (cached).

> #Using cached local data for templates

Continuing with my crusade against the word "cached," I think just removing the word here would be sufficient.